### ISSUE TITLE
Can't make new actors

### ISSUE THREAD

**Issue by 4lbburrito** [kypvalanx/Foundry-VTT-StarWars-SagaEdition#48]
So when I create a new actor no character sheet comes up. When I click on the icon or name of the actor nothing happens. Can anyone else reproduce?

**Comment by marvinandroid01** [comment]
I have a similar issue.  I am able to create characters and NPCs, but when double-clicking on the icon no character sheet comes up.  Right-clicking on the character brings up the usual menu (delete, duplicate, configure permissions, export data, import data), and when I export data it looks like the data is there, but the character sheet isn't coming up.  I tried attaching the JSON to this comment, but it wasn't attaching; I changed the suffix to .txt to attach it here.
[fvtt-Actor-test.txt](https://github.com/kypvalanx/Foundry-VTT-StarWars-SagaEdition/files/7191018/fvtt-Actor-test.txt)
